src/tir/schedule/ir_comparator.cc

```diff
@@ -125,9 +125,6 @@ bool TensorizeComparator::VisitStmt_(const BlockNode* op, const Stmt& other) {
     if (!CompareArray(op->iter_vars, rhs->iter_vars, &TensorizeComparator::CompareIterVar)) {
       return false;
     }
-    if (!CompareAnnotationMap(op->annotations, rhs->annotations)) {
```

Comment:
cc @vinx13 if this is ok. I cannot get anything working without removing this check.

Reply:
makes sense to me